<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addclientelement</class>
 <widget class="QDialog" name="addclientelement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>addClientElement</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Imie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_imie_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nazwisko_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adres_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nrtel_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adres</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr Telefonu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dodajKlienta_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj rekord</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
